src/components/modal/email-confirmation/modal.scss

```diff
@@ -0,0 +1,45 @@
+@import "../../../colors";
+@import "../../../frameless";
+
+
+.modal-content{
+    border-radius: 4px;
```

Comment:
I think this changed the way other modals on the project page look.

White corners on the Copy Link and Report modals:
<img width="520" alt="Screenshot 2022-01-15 at 8 00 59 PM" src="https://user-images.githubusercontent.com/83728060/149628634-98abe2db-d694-4e51-8862-197d9326b29b.png">

4px border radius on the Add to Studio modal:
<img width="522" alt="radius" src="https://user-images.githubusercontent.com/83728060/149628884-a283c3aa-950c-4ad4-a683-0ecaf46b75fc.png">

Reply:
Thanks @BarelySmooth!

CC/ @tomlum @seotts